python/cudf/cudf/groupby/groupby.py

```diff
@@ -44,10 +45,19 @@ def count(self):
     def agg(self, func):
         return self._apply_aggregation(func)
 
+    def size(self):
+        nrows = len(self._groupby.obj)
+        ones = cudf.Series(cudautils.ones(nrows, dtype="int8"))
```

Comment:
Can this just be an uninitialized column for efficiency?

Reply:
yeah, it can